export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+"""
+This script decomposes all static meshes into sets of convex regions.
+Decomposition method and parameters are based on the parameters chosen in UE, which are captured in 
+gltf_scene/actors_information.json.
+"""
+import argparse
+from collections import deque, defaultdict
+import copy
+import json
+import multiprocessing as mp
+if mp.current_process().name == 'MainProcess':
```

Comment:
- When is this false? `mp.current_process().name == 'MainProcess'`
- Find a better name for `utils.py` like `pipeline_utils.py` or `data_pipeline_utils.py`
- Our existing Python conventions: 4 spaces per indent, prefer `""` instead of `'` (consistency matters because it makes it easier to search for string literals)

- Move the weird `import` below the normal imports.

```
import argparse
from collections import deque, defaultdict
import copy
import json
import multiprocessing as mp
import numpy as np
import os
import shutil
import subprocess
import trimesh
import utils
import yaml

# coacd needs to be imported after bpy, otherwise terminate is called after throwing an std::bad_cast exception
if mp.current_process().name == "MainProcess":
    import bpy
import coacd
```